src/hotspot/share/gc/g1/g1FullGCScope.cpp

```diff
@@ -75,3 +81,7 @@ G1FullGCTracer* G1FullGCScope::tracer() {
 G1HeapTransition* G1FullGCScope::heap_transition() {
   return &_heap_transition;
 }
+
+size_t G1FullGCScope::hr_live_words_threshold() {
```

Comment:
Not sure about the name, but can't come up with something a lot better, maybe `region_compaction_threshold()` and a comment explaining that if the amount of live data is greater than this value the region won't be compacted.

Reply:
+1